apps/storybook/src/WhereToStart.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { WhereToStartSection } from '@asap-hub/react-components';
+
+export default {
+  title: 'Organisms / Dashboard / Where To Start',
+};
+
+export const Normal = () => <WhereToStartSection pages={[]} />;
```

Comment:
This is unfortunately useless for the preview, can you add a configurable number of pages here? (using something like `Array.from({length: number('Number of Pages', 2, {min: 0})}, () => {...})`, we have this in places)